component-samples/demo/aio/docker-compose.yml

```diff
@@ -19,7 +19,8 @@ services:
         - _JAVA_OPTIONS
     restart: on-failure:5
     env_file:
-      - ./aio.env
+      - aio.env
+      - creds.env
```

Comment:
++ Updating the new creds information & location in READEME.

[For raising an issue, so that we don't lose track of it.]